src/modules/pokeCounter.js

```diff
@@ -0,0 +1,9 @@
+export default async function pokeCounter() {
+  fetch('https://pokeapi.co/api/v2/pokemon?limit=9')
+    .then((response) => response.json())
+    .then((allpokemon) => {
+      const counter = allpokemon.results.length;
```

Comment:
We need to use the same function that you used in your `pokimonCounter` test,
I can see its the same logic, but also it needs to be the same function used.. :ok_hand: